packages/dappmanager/src/modules/migrations/setDefaultFeeRecipient.ts

```diff
@@ -0,0 +1,53 @@
+import { ConsensusClient, Network } from "@dappnode/common";
+import * as db from "../../db/index.js";
+import { packageGet } from "../../calls";
+import { listPackageNoThrow } from "../docker/list/listPackages.js";
+import { getValidatorServiceName } from "../stakerConfig/utils.js";
+
+/**
+ * Sets the default global environment variable FEE_RECIPIENT_<NETWORK>
+ * This global env var is used by the DAppNode packages to set the default fee recipient
+ */
+export async function setDefaultFeeRecipient(): Promise<void> {
+  const networks: Network[] = [];
+
+  if (db.feeRecipientPrater.get() === null) networks.push("prater");
+  if (db.feeRecipientGnosis.get() === null) networks.push("gnosis");
+  if (db.feeRecipientMainnet.get() === null) networks.push("mainnet");
+
+  for (const network of networks) {
+    const envVarName = `FEE_RECIPIENT_ADDRESS`;
```

Comment:
Why is this FEE_RECIPIENT_ADDRESS? Above you used FEE_RECIPIENT_<NETWORK> (no "ADDRESS"). Can this be problematic?

Reply:
It is the env var name used in clients compose files (i.e https://github.com/dappnode/DAppNodePackage-teku-prater/blob/6da01836c0dc395e90e25381d36913d3802c6558/docker-compose.yml#L13)